docs/hyn/5.4/laravel-scheduler.md

```diff
@@ -0,0 +1,229 @@
+
+```php
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Hyn\Tenancy\Contracts\Repositories\WebsiteRepository;
+use Hyn\Tenancy\Database\Connection;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Symfony\Component\Console\Exception\RuntimeException;
+
+class ExampleCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'example:command {--website_id=}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Example Command - Tenant Aware';
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * @var WebsiteRepository
+     */
+    private $websites;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->websites = app(WebsiteRepository::class);
+        $this->connection = app(Connection::class);
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $website_id = $this->option('website_id');
+        try{
+            $website = $this->websites->query()->where('id', $website_id)->firstOrFail();
+            $this->connection->set($website);
+            $this->info('Running Command on website_id: ' . $website_id);
+
+            $this->tenantHandle();
+
+            $this->connection->purge();
+        } catch (ModelNotFoundException $e) {
+            throw new RuntimeException(
+                sprintf(
+                    'The tenancy website_id=%d does not exist.',
+                    $website_id
+                )
+            );
+        }
+    }
+
+    /**
+     * Execute the console command. Tenant Aware.
+     *
+     * @return mixed
+     */
+    public function tenantHandle()
+    {
+        // You are now Tenant Aware
+        // Execute something, dispatch a Job, or anything else
+    }
+}
+```
+
+# Integration
+
+>This section of the tutorial assumes that you have successfully created one or more tenant aware commands.
+
+We now need to register our `ExampleCommand` into our schedule. Laravel's Scheduler is set into `app/Console/Kernel.php`.
+
+1. First, we need to tell Laravel which commands will be used in the scheduler
+
+```php
+protected $commands = [
+    ExampleCommand::class,
+    MySecondCommand::class,
+    ...
+];
+
+```
+
+2. Create a method for each commands that needs to run. We will be using the `before` [Task Hook](https://laravel.com/docs/master/scheduling#task-hooks) to specify code to be executed before the scheduled task is launched. Here is an example:
+
+```php
+/*
+ * Example Command Schedule
+ * @param  Illuminate\Console\Scheduling\Schedule   $schedule   Instance of schedule
+ * @return void
```

Comment:
Oke this is really OCD from me, but can you align this docblock format with the one a bit further down the page?

So:

```
/*
 * {DESCRIPTION}
 *
 * {PARAM}
 *
 * {RETURN}
 */
```